components/footer.jsx

```diff
@@ -23,6 +23,10 @@ const FOOTER_LINKS = [
   {
     link: `https://www.mozilla.org/privacy/websites`,
     text: `Privacy`
+  },
+  {
+    link: `https://www.mozilla.org/en-US/privacy/websites/#cookies`,
```

Comment:
though MozFest site isn't localized I think removing `en-US` from the URL and let mozilla.org shows localized content can still be good